src/main/java/net/imagej/ops/features/sets/FirstOrderStatsComputerSet.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
should we rather autogenerated those interfaces? they can live in the featureset() namespace?